docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,98 @@ If we compare the different fits, we can see that the Lorentzian model with diod
 While this is true for this particular dataset, no general statement can be made about the bias of fitting a Lorentzian rather than the hydrodynamically correct power spectrum.
 If low bias is desired, one should use the hydrodynamically correct model.
 On regular sensors, it is best to fit the hydrodynamically correct model with the diode model enabled.
+
+Active Calibration
+------------------
+
+Sometimes, more accuracy is required.
+One way to achieve this is with active force calibration.
+By providing a known perturbation to the bead, active calibration uses fewer assumptions than passive calibration.
+
+When performing passive calibration, we rely on knowledge of the drag coefficient to obtain a trap stiffness and positional calibration.
+This theoretical drag coefficient depends on parameters that are only known with limited precision.
+These parameters are the diameter of the bead and the viscosity.
+This viscosity in turn depends strongly on the local temperature around the bead, which is typically also poorly known.
+
```

Comment:
Paragraph 1 and 3 I fully agree.

For paragraph two I would rephrase it very slightly, because the drag coefficient is not estimated (which is actually what you indirectly do in active), but assumed, which is actually the problem.

What do you think of the following for paragraph 2?
```
When performing passive calibration, we base our calculations on a theoretical drag coefficient.
This theoretical drag coefficient depends on parameters that are only known with limited precision: the diameter of the bead and the viscosity.
This viscosity in turn depends strongly on the local temperature around the bead, which is typically poorly known.
```

Reply:
Sounds good!

And on a second though, I believe the first sentence of the 3rd paragraph should be: During active calibration, the trap or nanostage _is_ oscillated sinusoidally.

Reply:
Ah yes, good catch! Fixing...